### DISCUSSION TITLE
parcel v2 monorepos: proxy / alias not working

### DISCUSSION THREAD

**Discussion by csenio** [parcel-bundler/parcel#4924] [General]
I have a monorepo setup with parcel v2 & yarn workspaces. Proxy and Aliases didn't work out of the box, so I thought I'd explain how. I got them to work in case anyone else needs it:

1. alias
Couldn't get the native aliases or the "paths" section in tsconfig to work at all, instead I used a custom babelrc and https://github.com/tleunen/babel-plugin-module-resolver. 

2. proxy
proxyrc needs to be in the root of the monorepo (not the root of your proxy folder)

**Comment by mischnic** [comment]
> Couldn't get the native aliases

Did you put the alises in the subpackage's `package.json` and the npm package is actually installed in `node_modules/foo` in the monorepo root (because it was hoisted by Yarn)? This is a problem that I have run into myself

> proxy

We should put that in the docs (or change it)...

**Comment by dmitrijs-balcers** [comment]
> proxyrc needs to be in the root of the monorepo (not the root of your proxy folder)

@jescowuester thank you! You saved me bunch of time
